_includes/badges-bar.html

```diff
@@ -0,0 +1,17 @@
+<div class="badges-bar">
+  <a href="https://pypi.python.org/pypi/Scrapy">
+    <img alt="PyPI Version" src="https://camo.githubusercontent.com/aa264cdf106775e2ede947764e3cf3bb79b8fff4/68747470733a2f2f696d672e736869656c64732e696f2f707970692f762f5363726170792e737667" data-canonical-src="https://img.shields.io/pypi/v/Scrapy.svg" style="max-width:100%;">
+  </a>
+  <a href="http://travis-ci.org/scrapy/scrapy">
+    <img alt="Build Status" src="https://camo.githubusercontent.com/dbf9a922328ddcf26f3b97c54a8db5c826d685da/68747470733a2f2f696d672e736869656c64732e696f2f7472617669732f7363726170792f7363726170792f6d61737465722e737667" data-canonical-src="https://img.shields.io/travis/scrapy/scrapy/master.svg" style="max-width:100%;">
+  </a>
+  <a href="https://pypi.python.org/pypi/Scrapy">
+    <img alt="Wheel Status" src="https://camo.githubusercontent.com/609890facf66fde6b5065196cde85ed95efa1c43/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f776865656c2d7965732d627269676874677265656e2e737667" data-canonical-src="https://img.shields.io/badge/wheel-yes-brightgreen.svg" style="max-width:100%;">
+  </a>
+  <a href="https://github.com/scrapy/scrapy/wiki/Python-3-Porting">
+    <img alt="Python 3 Porting Status" src="https://camo.githubusercontent.com/4435e82dc7497e4e896e70483c0233ea3df6ce5d/687474703a2f2f7374617469632e7363726170792e6f72672f70793370726f67726573732f62616467652e737667" data-canonical-src="http://static.scrapy.org/py3progress/badge.svg" style="max-width:100%;">
+  </a>
+  <a href="http://codecov.io/github/scrapy/scrapy?branch=master">
+    <img alt="Coverage report" src="https://camo.githubusercontent.com/e8aa0afb265628547b91b7fd8b375cb055960860/68747470733a2f2f696d672e736869656c64732e696f2f636f6465636f762f632f6769746875622f7363726170792f7363726170792f6d61737465722e737667" data-canonical-src="https://img.shields.io/codecov/c/github/scrapy/scrapy/master.svg" style="max-width:100%;">
```

Comment:
`src` attribute is using GH content distribution network, the real link must be the one in `data-canonical-src` attribute.